payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/DropdownFieldUI.kt

```diff
@@ -61,12 +61,11 @@ internal fun DropDown(
     controller: DropdownFieldController,
     enabled: Boolean,
 ) {
-    val label by controller.label.collectAsState(
-        null
-    )
+    val label by controller.label.collectAsState(null)
     val selectedIndex by controller.selectedIndex.collectAsState(0)
     val items = controller.displayItems
     var expanded by remember { mutableStateOf(false) }
+    val selectedItemLabel = controller.getSelectedItemLabel(selectedIndex)
```

Comment:
Are items and selectedIndex replaced by selectedItemLabel?

Reply:
I replaced line 114 `items[selectedIndex]` on the old version with `selectedItemVersion` in line 101 and 126 here. Is that what you mean?